### PR TITLE
[NEW BONUS GUIDE] Install / Update / Uninstall common languages

### PR DESCRIPTION
**Pull request by satsHIVE**
#### What

As discussed with @VajraOfIndra, this is a section in the guides to collect all the installations of common prerequisites across the other guides. 

### Why

Central management of the guides will allow easier maintenance without the need to adapt individual application and tools guides.

#### How

Prepare guides for the common languages across the guides and collect them into one section.

#### Scope

- [ ] significant change to core configuration
- [x] independent bonus guide
- [ ] simple bug fix

Fixes # (link issue)

#### Test & maintenance

Only the language guides will be updated on new versions of the languages or changes in the overall process. Tools and app guides shall not be touched.

#### Animated GIF (optional)

n/a

### CODE REVIEW

**Comment by blckbx** [comment]
@VajraOfIndra @kristapsk 

Just a thought for every guide in which checksums are NOT verified automatically, so you have to manually verify the checksum char by char. Instead of doing so, can we do something like this?
```sh
$ echo "99de2fe112a52ab748fb175edea64b313a0c8d51d6157dba683a6be163fd5eab go1.19.3.linux-arm64.tar.gz" | sha256sum --check
go1.19.3.linux-arm64.tar.gz: OK
```
Example is taken from this PR.

File: guide/bonus/lightning/regolancer.md [hunk 1]
```diff
@@ -42,52 +42,7 @@ Table of contents
 
 ## Install GO
```
Comment (VajraOfIndra) [inline guide/bonus/lightning/regolancer.md:43]:
```suggestion
## Install Go
```

**Review by VajraOfIndra — CHANGES_REQUESTED** [review]
I really like the idea of starting to provide these languages guides 💯 
I think they might help us make the other guides clearer and reduce the number of issues users might encounter with outdated language versions or confusions with the various ways of installing or updating a language.

_In term of this specific Go PR:_
LGTM, maybe a couple of needed changes (the sudo afaik) + a few nits below (feel free to keep the ones you like and ignore the ones you don't).

**Review by VajraOfIndra — APPROVED** [review]
LGTM, tACK!

**Comment by VajraOfIndra** [comment]
> Just a thought for every guide in which checksums are NOT verified automatically, so you have to manually verify the checksum char by char. Instead of doing so, can we do something like this?
> 
> ```shell
> $ echo "99de2fe112a52ab748fb175edea64b313a0c8d51d6157dba683a6be163fd5eab go1.19.3.linux-arm64.tar.gz" | sha256sum --check
> go1.19.3.linux-arm64.tar.gz: OK
> ```

Sounds like a nice solution yes..

@satsHIVE would you like to implement that change in your guide before we merge it..?

**Comment by satsHIVE** [comment]
Yes, will do. But can in a few hours if not an issue.

**Comment by satsHIVE** [comment]
sha256sum aligned to suggestion. @VajraOfIndra @blckbx

File: guide/bonus/raspberry-pi/golang.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+  ```sh
+  $ cd /tmp
+  $ wget https://go.dev/dl/go1.19.3.linux-arm64.tar.gz
+  ```
+
+* Check on the download page what is the SHA256 checksum of the file, e.g. for the above:
+99de2fe112a52ab748fb175edea64b313a0c8d51d6157dba683a6be163fd5eab. Calculate the SHA256 hash of the downloaded file. It should give an "OK" as an output.
+
+  ```sh
+  $ $ echo "99de2fe112a52ab748fb175edea64b313a0c8d51d6157dba683a6be163fd5eab go1.19.3.linux-arm64.tar.gz" | sha256sum --check
```
Comment (VajraOfIndra) [inline guide/bonus/raspberry-pi/golang.md:44]:
```suggestion
  $ echo "99de2fe112a52ab748fb175edea64b313a0c8d51d6157dba683a6be163fd5eab go1.19.3.linux-arm64.tar.gz" | sha256sum --check
```

File: guide/bonus/lightning/circuit-breaker.md [hunk 3]
```diff
@@ -41,52 +41,7 @@ Table of contents
...
-  $ source /etc/profile
-  ```
-
-* Test that Go has been properly installed by checking its version
-
-  ```sh
-  $ go version
-  > go version go1.17.8 linux/arm64
-  ```
+* To [install Go](https://raspibolt.org/guide/bonus/raspberry-pi/golang.html#install-go) follow the instructions provided in the bonus guide.
```
Comment (VajraOfIndra) [inline guide/bonus/lightning/circuit-breaker.md:44]:
```suggestion
* To [install Go](../raspberry-pi/golang.md#install-go) follow the instructions provided in the bonus guide.
```

File: guide/bonus/lightning/regolancer.md [hunk 4]
```diff
@@ -40,54 +40,9 @@ Table of contents
...
-  $ source /etc/profile
-  ```
-
-* Test that Go has been properly installed by checking its version
-
-  ```sh
-  $ go version
-  go version go1.19.2 linux/arm64
-  ```
+* To [install Go](https://raspibolt.org/guide/bonus/raspberry-pi/golang.html#install-go) follow the instructions provided in the bonus guide.
```
Comment (VajraOfIndra) [inline guide/bonus/lightning/regolancer.md:45]:
```suggestion
* To install Go follow the instructions provided in the [bonus guide](../raspberry-pi/golang.md#install-go).
```

**Comment by VajraOfIndra** [comment]
Let's go to master! 🎉

**Pull request merged**